### ISSUE TITLE
css changes in Bootstrap 5

### ISSUE THREAD

**Issue by gyver-mac** [gantry/gantry5#2853]
First image : J3.9 (bootstrap 2.xx) 
Second image : J4.0 (bootstrap 5.xx)

As you can see the background-color is different !
Here inside .platform-content the class "moduletable" has been replaced by "card" and "card" has background-color:#fff with bootstrap 5.

Not really a bug but a lot of work with templates   :-(

![Capture d’écran 2021-08-18 à 14 55 03](https://user-images.githubusercontent.com/88447159/129901786-cfd2406e-793a-4411-9c05-b025aacb97bb.png)

![Capture d’écran 2021-08-18 à 14 55 20](https://user-images.githubusercontent.com/88447159/129901934-53662c9c-32db-4281-941b-1459fbef6d6d.png)

**Comment by mahagr** [comment]
Yeah.. I think there's somewhere some migration guide, maybe take a look into the Bootstrap docs?

**Comment by gyver-mac** [comment]
Yes there's migration guides ... 
From Bootstrap 2.3 to 3 then 3 to 4 and 4 to 5 !!!

**Comment by hexplor** [comment]
Do you have somewhere a page with that active issue? Maybe we can find a way to solve it quickly.

**Comment by gyver-mac** [comment]
Yes you can go to : [https://www.universalmac.interface34.fr/expertise](url)

**Comment by mahagr** [comment]
I think this may be related to Joomla 4 changing the module classes. See #2873 and https://github.com/gantry/gantry5/blob/feature/v5.5.0/Joomla4-support.md
